lib/iris/fileformats/netcdf.py

```diff
@@ -84,7 +84,7 @@
              'coordinates', '_FillValue', 'formula_terms', 'grid_mapping',
              'leap_month', 'leap_year', 'long_name', 'missing_value',
              'month_lengths', 'scale_factor', 'standard_error_multiplier',
-             'standard_name', 'units', 'valid_max', 'valid_min', 'valid_range']
```

Comment:
> they are not interpreted and they are simply thrown away on load

Ah. Well in that case these attributes definitely should **not** be in this list!

> Perhaps the issue here is the variable name `_CF_ATTRS` not its contents

Correct; that was what threw me. To me it sounded like these were controlled _and processed_ attributes, not controlled and ignored attributes. Happy to stand down on this one!